categories.html

```diff
@@ -0,0 +1,509 @@
+                        <label>
+                            <span class="row-data">Python</span>
+                            <span class="row-data-num">( 4,252 )</span>
+                        </label>
+                    </div>
+                </div>
+            </div>
+
+            <!-- aside level -->
+            <div class="aside-container">
+                <h3>Level</h3>
+                <div class="aside-content">
+                    <div class="aside-row">
+                        <input type="checkbox" name="" id="">
+                        <label>
+                            <span class="row-data">All Levels</span>
+                            <span class="row-data-num">( 4,252 )</span>
+                        </label>
+                    </div>
+
+                    <div class="aside-row">
+                        <input type="checkbox" name="" id="">
+                        <label>
+                            <span class="row-data">Beginner</span>
+                            <span class="row-data-num">( 4,252 )</span>
+                        </label>
+                    </div>
+                    <div class="aside-row">
+                        <input type="checkbox" name="" id="">
+                        <label>
+                            <span class="row-data">Intermediate</span>
+                            <span class="row-data-num">( 4,252 )</span>
+                        </label>
+                    </div>
+                    <div class="aside-row">
+                        <input type="checkbox" name="" id="">
+                        <label>
+                            <span class="row-data">Expert</span>
+                            <span class="row-data-num">( 4,252 )</span>
+                        </label>
+                    </div>
+                </div>
+            </div>
+
+            <!-- aside price -->
+            <div class="aside-container">
+                <h3>Price</h3>
+                <div class="aside-content">
+                    <div class="aside-row">
+                        <input type="checkbox" name="" id="">
+                        <label>
+                            <span class="row-data">Free</span>
+                            <span class="row-data-num">( 4,252 )</span>
+                        </label>
+                    </div>
+
+                    <div class="aside-row">
+                        <input type="checkbox" name="" id="">
+                        <label>
+                            <span class="row-data">Paid</span>
+                            <span class="row-data-num">( 4,252 )</span>
+                        </label>
+                    </div>
+                </div>
+            </div>
+        </aside>
+        <!-- main content -->
+        <div class="main-content">
+            <div class="card">
+                <img src="./images/homepage/webdev.png" alt="product">
+                <div class="card-container">
```

Comment:
In image you can use --> loading=lazy <-- . It will help in the optimization of your app.